### PR TITLE
Eslinted

### PR DESCRIPTION
**Pull request by jamesb8891**

### CODE REVIEW

File: user-stories.md [hunk 1]
```diff
@@ -9,11 +8,8 @@
...
 
-4. There should be a separate input that I can search multiple countries to have them live on the page and compare them. 
+4. There should be a separate input that I can search multiple countries to have them live on the page and compare them.
 
 5. As a user I should be able to star a country and have it saved to local storage. I should be able to view my starred countries on page reload. I should be able to unstar my favorite countries so they don't persist on page reload.
 
-6. 
-
--add flags to each country
--
```
Comment (Dhanciles) [inline user-stories.md]: I think we should delete this file all from our directory.

**Comment by Dhanciles** [comment]
This looks great James, thank you for doing this! Check out my comment for deleting the user story file and I think we'll be good to merge after that.

File: user-stories.md (hunk 1 above)
Comment (colev1) [reply user-stories.md]: Agreed

Comment (jamesb8891) [reply user-stories.md]: Will do!

**Comment by colev1** [comment]
Looks good now, I'll merge

**Pull request merged**